logstash-core/lib/logstash/compiler/treetop_monkeypatches.rb

```diff
@@ -90,4 +90,16 @@ def _inspect(indent="")
         ""
       )
   end
+
+  def _parse_context(context = self)
```

Comment:
style: I was curious about the `_` method name prefix, but I found only one prior method here (`_inspect)` which I assume was named to avoid clashing with Ruby's `Object#inspect` ?

Can you tell me about the reason for the underscore prefix?

Reply:
yes; we're monkey-patching, and avoiding collisions is the goal.